Frontend/src/app/query-interface/page.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import React, { useState, useEffect }  from "react";
+import { Input } from 'antd';
+import { DownloadOutlined, CaretRightOutlined } from '@ant-design/icons';
+
+const { TextArea } = Input;
+
+export default function Query() {
+  const [value, setValue] = useState<string>('');
+
+  return (
+    <div className="">
+      <div style={{display: "flex", justifyContent: "space-between", width: "100%", border: "1px solid #d9d9d9", borderRadius: "8px", padding: "10px", marginTop: "15px"}}>
+        <div style={{width: "20px", margin: "2px"}}>
+          $:
+        </div>
+        <div style={{width: "90%"}}>
+          <TextArea 
+            placeholder="Query ..." 
```

Comment:
Can we use better message than "Query ..."?

Reply:
like what ??

Reply:
Lets have a call and discuss the message.